packages/core/lib/segment_emitter.js

```diff
@@ -1,12 +1,70 @@
 var dgram = require('dgram');
 
+var batcher = require('atomic-batcher');
 var logger = require('./logger');
 
 var DEFAULT_ADDRESS = '127.0.0.1';
 var DEFAULT_PORT = 2000;
 var PROTOCOL_HEADER = '{"format": "json", "version": 1}';
 var PROTOCOL_DELIMITER = '\n';
 
+function batchRun (work, callback) {
```

Comment:
You have a variable named `work` defined in line46 which is a dictionary contains ip and port etc. Is this work the same here? Should it be the actual `ops`?

Reply:
Good catch! The name `ops` sounds good to me.